dev-python/typer/typer-0.15.3.ebuild

```diff
@@ -0,0 +1,92 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{11..13} )
+DISTUTILS_USE_PEP517=pdm-backend
+
+inherit distutils-r1 shell-completion pypi
+
+DESCRIPTION="Build great CLIs. Easy to code. Based on Python type hints."
+HOMEPAGE="
+	https://typer.tiangolo.com/
+	https://github.com/tiangolo/typer
+	https://pypi.org/project/typer/
+"
+SRC_URI="$(pypi_sdist_url "${PN}" "${PV}")"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~arm64"
+
+IUSE="+rich shell-detection test"
+
+BDEPEND="
+	test? (
+		$(python_gen_cond_dep '
+			dev-python/coverage[${PYTHON_USEDEP}]
+			dev-python/shellingham[${PYTHON_USEDEP}]
+		')
+	)
+"
+RDEPEND="
+	$(python_gen_cond_dep '
+		dev-python/click[${PYTHON_USEDEP}]
+		dev-python/typing-extensions[${PYTHON_USEDEP}]
+		rich? ( dev-python/rich[${PYTHON_USEDEP}] )
+		shell-detection? ( dev-python/shellingham[${PYTHON_USEDEP}] )
+	')
+"
+
+distutils_enable_tests pytest
+
+src_test() {
+	export TERMINAL_WIDTH=3000
+	export _TYPER_FORCE_DISABLE_TERMINAL=1
+
+	distutils-r1_src_test "${@}"
+}
+
+src_compile() {
+	distutils-r1_src_compile "${@}"
+
+	local shell
+	for shell in bash zsh fish; do
+		typer_gencomp ${shell}
```

Comment:
You can't do that in `src_compile`, there is no valid Python set here; only in `python_compile`.

Reply:
I think it's fixed now.